tools/topology/topology2/include/components/virtual_widget.conf

```diff
@@ -0,0 +1,46 @@
+#
+# SOF virtual widget
+#
+# A generic volume component. All attributes defined herein are namespaced
```

Comment:
This looks like a copy paste from volume component. Can you add the correct comment for virtual widget - I assume it maps to the ASoC virtual widget ? if so , we should say so in the comments.

Reply:
thanks